agent/logger/field/constants.go

```diff
@@ -35,4 +35,5 @@ const (
 	Image         = "image"
 	Volume        = "volume"
 	Time          = "time"
+	NetworkMode   = "networkMode"
```

Comment:
Why is this part of `logger`?

Reply:
This is so that I could add the following logging line in `initNetworkMode`:
```
	logger.Info("Task network mode initialized", logger.Fields{
		field.TaskID:      task.GetID(),
		field.NetworkMode: aws.StringValue(acsTaskNetworkMode),
	})
```